BUILDING.md

```diff
@@ -13,7 +13,68 @@ For building the manual, see the README in the [Manual repository](https://githu
 Windows
 -------
 
-    TODO
+You should have some experience in working with console commands, like changing directories
+running commands aso.
+
+### Visual studio version ###
```

Comment:
It's written in the guide to open the command prompt. So it should be obvious.

Reply:
It is obvious when you have read the whole section. The title is read before that.

Reply:
Dropped this paragraph. I thought it might help Windows users estimating the required expertise. But probably github users are competent enough and info like this is not needed.